backend/src/test/java/com/woowacourse/levellog/acceptance/AcceptanceTest.java

```diff
@@ -63,5 +88,112 @@ public void setUp(final RestDocumentationContextProvider contextProvider) {
                                         prettyPrint()
                                 )
                 ).build();
+
+        masterToken = getToken(login(MASTER));
```

Comment:
```suggestion
        masterToken = getToken(Objects.requireNonNull(login(MASTER)));
```
인텔리제이에서 `login()`에서 null 반환이 될 수 있어 검증 추가를 추천해주는데 테스트 상 null을 리턴되지 않도록 구현했기 때문에 그대로 둔걸까요?